src/test/java/com/productdock/library/inventory/adapter/out/kafka/KafkaProducerShould.java

```diff
@@ -12,7 +12,7 @@
 import static org.mockito.Mockito.mock;
 
 @ExtendWith(MockitoExtension.class)
-class KafkaProducerUnitTest {
+class KafkaProducerShould {
```

Comment:
Check test names.
E.g.: `produceCorrectMessage` -> `produceMessage`, `throwExceptionWhenProducingBadEntity`  -> `throwExceptionWhenReceivedBadMessage`

Reply:
produceMessage doesn't describe if the message is correct or not, which is important for the test

Reply:
throwExceptionWhenReceivedBadMessage is misleading, we are testing producing of a message here

Reply:
Discussion resolved